dbt_subprojects/daily_spellbook/models/_sector/utils/utils_months_table.sql

```diff
@@ -0,0 +1,18 @@
+{{ config(
+    schema = 'utils',
+    alias = 'months_table',
+    materialized = 'table',
+    file_format = 'delta'
+    )
+}}
+-- these are set up as tables instead of incremental models because this guarantees the ordering and compute is negligable.
+
+SELECT timestamp
+FROM unnest(
+    sequence(
+        timestamp '2009-01-03'
+        , cast(date_trunc('month', now()) as timestamp)+ interval '3' day  -- add some padding to account for materialization lag
+        , interval '1' month
```

Comment:
@hildobby you should add the materialization padding inside the `date_trunc` here.
Same for weeks and years.